src/Microsoft.AspNetCore.Mvc.WebApiCompatShim/HttpError.cs

```diff
@@ -19,6 +20,9 @@ namespace System.Web.Http
     [XmlRoot("Error")]
     public sealed class HttpError : Dictionary<string, object>, IXmlSerializable
     {
+        // Silly spelling of empty1111 to avoid collisions with other ModelState entries.
+        private static readonly string EmptyKey = "\x00C9m\x00FEt\x00DD1111";
```

Comment:
Could we limit this change to `SerializableError` (i.e. not update WebApiCompatShim)?

Reply:
We could, later. For now `HttpError` is a supported class and I've done the work to fix this issue and the nested-deserialization bug there.

Reply:
@mkArtakMSFT thoughts on this?

Reply:
Per offline conversation, I'll remove `HttpError`-related bits.